### ISSUE TITLE
Build error when upgrading version fro v8.0.0-beta.8 to v8.3.1

### ISSUE THREAD

**Issue by gonzalochief** [go-redis/redis#1536]
Issue tracker is used for reporting bugs and discussing new features. Please use
[stackoverflow](https://stackoverflow.com) for supporting issues.

I was using go-redis without issues until i updated my golang mod files and i started getting this build error:
cannot find package "go.opentelemetry.io/otel/apils" in any of: /usr/local/go/src/go.opentelemetry.io/otel/apils (from $GOROOT)
Iḿ currently using go version go1.15.2 linux/amd64.
The compiler gets all the opentelemetry package but the apils package

## Expected Behavior

The program should compile without error

## Current Behavior

cannot find package "go.opentelemetry.io/otel/apils" in any of: /usr/local/go/src/go.opentelemetry.io/otel/apils (from $GOROOT)

## Possible Solution

Update the code using trace.### to point the otel package

## Steps to Reproduce

1.Create a clean development environment
2. Clone a repo using go-redis
3. go build the code
4. The error appears

**Comment by vmihailenco** [comment]
https://github.com/go-redis/redis/issues/1534

**Comment by gonzalochief** [comment]
I checked the Otel source code and the trace package was moved to the otel package in v0.13.0. If you check the source code of the otel package, there is a trace.go file as part of the otel package.
The issue is that the source code of go-redis keeps importing the /otel/api/trace package. This causes the compiler to try get a package that does not exists anymore. see files: options.go, redis.go, util.go, and conn.go.
I hope this helps... i have not been able to compile my code due to this issue. As always, thanks for all your help. Let me know if there is anything i can do
